doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,459 @@
+    "\n",
+    "If the samples are normally distributed and the SEM is known, the upper and lower bound of the $95~\\%$ CI are $\\overline{X}\\pm1.96s$.\n",
+    "\n",
+    "### Interquartile range\n",
+    "The interquartile range denotes the range, within which the central $50~\\%$ of all samples lie, if one were to order them by their size. This leaves one quarter of all samples lying below the interquartile range, and another quarter of all samples above it.\n",
+    "\n",
+    "### Now &ndash; what do we use?\n",
+    "We are interested in the precision of our overall, averaged, simulation result, and not in the precision of the individual samples. Those are expected to fluctuate, and in many cases, those fluctuations are uninteresting for the end result. Out of the options presented above, the SEM and the CI are the only ones doing this requirement justice. Since they are related, the question boils down to how to compute the SEM, which will be the topic of the rest of this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncorrelated samples\n",
+    "How the SEM can be computed depends on the data itself. For uncorrelated samples, it is nearly trivial:\n",
+    "\n",
+    "$\n",
+    "\\begin{align}\n",
+    "    \\hat\\sigma_\\overline{X} = \\frac{\\hat\\sigma}{\\sqrt{N}}\\tag{2}\n",
+    "\\end{align}\n",
+    "$\n",
+    "\n",
+    "where $\\hat\\sigma_\\overline{X}$ is the estimated SEM, $\\hat\\sigma$ is the estimated standard deviation (see eq. 1) and $N$ is the number of samples. But what does it mean for samples to be uncorrelated?\n",
+    "\n",
+    "An example for uncorrelated samples would be the rolling of a dice. The outcome of each trial is completely independent to the previous trials. We might guess any number from 1 to 6, regardless of what has been the last result. The same could be true if we ran an experiment many times independently from one another and measured a quantity each time. By looking at one experimental value, we would'nt be able to predict the next one. The best guess would be simply the mean value of the entire series.\n",
+    "\n",
+    "However, usually, when you run a molecular dynamics simulation, the particles will only move by a tiny amount during a time step. Consequently, most observables also change only by a small amount during a time step. If we were to sample an observable in every time step, we would get a lot of samples with very similar values. It is said that the samples are *correlated*. Only if we wait for a sufficiently long time, the system will eventually have evolved to a completely different configuration, and we can expect the observable to assume a truly independent, *uncorrelated* value.\n",
```

Comment:
```suggestion
    "Usually, when you run a molecular dynamics simulation, the particles will only move by a tiny amount during a time step. Consequently, most observables also change only by a small amount during a time step and it is, therefore, more probable to obtain a similar result rather than a completely different result. If we were to sample an observable in every time step, we would get a lot of samples with very similar values. It is said that the samples are *correlated*. Only if we wait for a sufficiently long time, the system will eventually have evolved to a completely different configuration, and we can expect the observable to assume a truly independent, *uncorrelated* value.\n",
```